### ISSUE TITLE
Quantum Computer

### ISSUE THREAD

**Issue by TheNain38** [HelloZeroNet/ZeroNet#185]
Quantum computer can break BitCoin cryptography
http://pqcrypto.org/

**Comment by HelloZeroNet** [comment]
If this happens then every currently known computer network become unsafe.

Currently this is only "in theory" and not reality. The encryption will evolve when Bitcoin's current cryptography breaking computers become reality and we will follow change.

**Comment by sbDevGit** [comment]
Yes, there will always be unsolvable mathematics equations on which the cryptography is based on.

**Comment by bashrc** [comment]
There are post-quantum crypto implementations, but as far as I know they're somewhat experimental and not yet shipped by default within Openssl/Libressl. I'd estimate that a government with a large budget could possibly build a quantum cryptanalysis machine for prime factoring small key lengths within a decade.

**Comment by HelloZeroNet** [comment]
The current bitcoin address format can be considered as quantum proof:
"quantum computing is not very useful for breaking hashes like SHA-256. Just signature and encryption algorithms like ECDSA or RSA. To attack ECDSA, you need to know the public key. However, most addresses are made from the hash of the public key, which is not enough to start attacking ECDSA. You'd need to break the hash first, which won't be significantly easier with quantum computers.
TL;DR any normal address that hasn't been spent from before is safe. So, if you use Bitcoin the way the developers recommend and don't reuse addresses, your coins are quantum safe, at least for now." - https://www.reddit.com/r/Bitcoin/comments/3wfmg2/satoshis_unmoved_coins_are_the_worlds_biggest/cxw6osp

**Comment by border0464111** [comment]
I don't think it's a zeronet specific issue and I don't thing it's even a real issue yet.
When Openssl will be declared unsecure anymore, It will be a good time to address the problem.   
Closing ?
